fixtures/dom/src/components/fixtures/input-change-events/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import FixtureSet from '../../FixtureSet';
+import TestCase from '../../TestCase';
+import RangeKeyboardFixture from './RangeKeyboardFixture';
+import RadioClickFixture from './RadioClickFixture';
+import InputPlaceholderFixture from './InputPlaceholderFixture';
+
+class InputChangeEvents extends React.Component {
+  render() {
+    return (
+      <FixtureSet
+        title="Input change events"
+        description="Tests proper behavior of the onChange event for inputs"
+      >
+        <TestCase
+          title="Range keyboard changes"
+          description={`
+            Range inputs should fire onChange events for keyboard events
+          `}
+        >
+          <TestCase.Steps>
+            <li>Focus range input</li>
+            <li>change value via the keyboard arrow keys</li>
+          </TestCase.Steps>
+
+          <TestCase.ExpectedResult>
+            The <code>onKeyDown</code> call count should be equal to
+            the <code>onChange</code> call count.
+          </TestCase.ExpectedResult>
+
+          <RangeKeyboardFixture />
+        </TestCase>
+
+        <TestCase
+          title="Radio input clicks"
+          description={`
+            Radio inputs should only fire change events when the checked
+            state changes.
+          `}
+          minimumVersion="16.0.0"
```

Comment:
I'd be happy to pick up some of the work here. @jquense if you are busy on other things, I've got time to work on this bit. Would a PR to your branch be fine?

Or if you wanna jam on it, that's fine too. I just wanted to pick up some of the load.